apps/condo/domains/resident/schema/ServiceConsumer.test.js

```diff
@@ -61,48 +64,67 @@ describe('ServiceConsumer', () => {
         })
     })
 
-    describe('SoftDelete', () => {
-        it('can be soft-deleted by user with type === resident if this is his own serviceConsumer', async () => {
-            const client1 = await makeClientWithServiceConsumer()
+    describe('Read', () => {
+        it('can be read by admin', async () => {
+            const adminClient = await makeLoggedInAdminClient()
+            await makeClientWithServiceConsumer()
 
-            const [deleted] = await updateTestServiceConsumer(client1, client1.serviceConsumer.id, { deletedAt: 'true' })
-            expect(deleted.id).toEqual(client1.serviceConsumer.id)
-            expect(deleted.deletedAt).toBeDefined()
+            const objs = await ServiceConsumer.getAll(adminClient, {}, { sortBy: ['updatedAt_DESC'] })
+
+            expect(objs.length >= 1).toBeTruthy()
+            expect(objs[0].id).toBeDefined()
```

Comment:
Can you rewrite test without `[0]`?
It will help us in moving to parallel test launching ;)

Reply:
Possible solution: add `where: {id / id_in}`

Reply:
Yep